themes/dark/assets/js/templates/add-show-options.vue

```diff
@@ -0,0 +1,258 @@
+<template>
```

Comment:
Notice you added that SFC to light and dark themes, it is no longer needed, so just remove the files and be sure to merge develop.

Reply:
I noticed your pr